vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java

```diff
@@ -2059,6 +2073,322 @@ private void createNotBufferedDynamicEditor() {
                 grid);
     }
 
+    // Drag and drop
+
+    private List<Person> draggedItems;
+    private Grid<Person> dragSource;
+
+    private void createRowReordering() {
+        // begin-source-example
+        // source-example-heading: Row Reordering
+
+        Grid<Person> grid = new Grid<>(Person.class);
+        grid.setColumns("firstName", "lastName", "phoneNumber");
+        List<Person> items = new PersonService().fetch(0, 5);
+        grid.setItems(items);
+
+        grid.setRowsDraggable(true);
+        grid.setDropMode(DropMode.BETWEEN);
+
+        grid.addDragStartListener(
+                event -> draggedItems = event.getDraggedItems());
+
+        grid.addDropListener(event -> {
+            if (event.getDropTargetRow().isPresent()) {
+                items.remove(draggedItems.get(0));
```

Comment:
This breaks when dragging something else than rows (no draggedItems)

Reply:
Disabled drop mode by default